### DISCUSSION TITLE
Change the speak rate for Tortoise

### DISCUSSION THREAD

**Discussion by victorcatalin** [rsxdalv/TTS-WebUI#190] [Q&A] [answered]
Hi, is there any chance to change the speaking rate in Tortoise? In my project i need the generation to speak slower, like for a story. In Edge TTS you can adjust the speed for the output, but i don't understand how to adjust the speed speaking rate for Tortoise. Much appreciated!

**Comment by rsxdalv** [comment, marked as answer]
Hi, I think the voice you select gives it a good baseline. For example "freeman" is a rather slow voice. Also you can make your own voices from mp3 files or using more advanced techniques to get a better result!

**Reply by victorcatalin** [reply]
I see. Thank you! And, again, thank you for all the hard work. Please maintain and update it, it's a lifesaver for us... <3 I hope you will succeed in everything!
